controller/controller.js

```diff
@@ -12,5 +12,78 @@ exports.getApiTopics = (request, response) => {
     });
 };
 
+// /api/articles Endpoints
+exports.getApiArticles = (request, response) => {
```

Comment:
There's too many comments on your code, you need to keep them short and precise so that they don't bloat your files!